packages/analytics/src/integrations/Omnitracking/__tests__/omnitracking-helper.test.ts

```diff
@@ -101,3 +102,21 @@ describe('getCheckoutEventGenericProperties', () => {
     });
   });
 });
+
+describe('getProductLineItemsQuantity', () => {
+  it('should return quantity equals to 10', () => {
```

Comment:
Nitpicking: When describing the test, describe what the logic should be instead of being too specific, something like this:

```suggestion
  it('should return the sum of all quantities from all products in the list', () => {
```

Both ways describe the same thing but the first one is more generic and I can understand that what it is doing is summing all the quantities more easily.

Reply:
Fixed